src/qt/peertablemodel.h

```diff
@@ -74,7 +74,33 @@ public Q_SLOTS:
 
 private:
     interfaces::Node& m_node;
-    const QStringList columns{tr("Peer Id"), tr("Address"), tr("Type"), tr("Network"), tr("Ping"), tr("Sent"), tr("Received"), tr("User Agent")};
+    const QStringList columns{
+        /*: Title of Peers Table column which contains a
+            unique number used to identify a connection. */
+        tr("Peer"),
+        /*: Title of Peers Table column which contains the
+            IP/Onion/I2P address we used to connect to the peer. */
```

Comment:
"we used to connect to the peer" -- looks like it is about outgoing connections only.

Reply:
what do you think about this rephrasing:
 `Title of the Peers Table column which contains the IP/Onion/I2P address of the connected peer`

Reply:
lgtm

Reply:
"column for the peer address" ?

(reviewing from my phone so apologies if a bit out of context)